src/js/pages/task-details/TaskDetail.js

```diff
@@ -139,10 +139,18 @@ class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
   }
 
   handleOpenLogClick(selectedLogFile) {
-    this.setState({selectedLogFile, currentTab: 'debug'});
+    let params = Object.assign(
+      {},
+      this.props.params,
+      {filePath: encodeURIComponent(selectedLogFile.get('path'))}
+    );
+    let currentRoutes = this.context.router.getCurrentRoutes();
+    let {logRouteName} = currentRoutes[currentRoutes.length - 1];
```

Comment:
How about `let {logRouteName} = currentRoutes.pop()` which is a little less error prone.